examples/fabric/build_your_own_trainer/trainer.py

```diff
@@ -4,11 +4,14 @@
 from typing import Any, cast, Iterable, List, Literal, Optional, Tuple, Union
 
 import torch
-from lightning_utilities.core import apply_to_collection, is_overridden
+from lightning_utilities import apply_to_collection, is_overridden
```

Comment:
I don't see a value in doing this. One of the main motivations for https://github.com/Lightning-AI/utilities/pull/120 was that we can make this import in one line. With your change, it would become two lines again.

Reply:
It simplifies the code as the parent doesnt need to be passed. I would say that is more important than how an import looks

Reply:
Could you send a PR with your suggestion? 😊